cfme/tests/cloud/test_keypairs.py

```diff
@@ -190,3 +186,25 @@ def test_download_private_key(keypair):
         initialEstimate: 1/4h
     """
     keypair.download_private_key()
+
+
+@pytest.mark.meta(automates=[1741635, 1747179])
+@test_requirements.multi_tenancy
+def test_tenantadmin_view_keypair(appliance, new_tenant_admin):
+    """
+    Test to verify that child tenants can see key pairs of parent tenants.
+
+    Polarion:
+        assignee: nachandr
+        casecomponent: Configuration
+        caseimportance: high
+        tags: cfme_tenancy
+        initialEstimate: 1/4h
+    """
+    view = navigate_to(appliance.collections.cloud_keypairs, 'All')
+    key_pair = view.entities.get_first_entity().data['name']
```

Comment:
Why not get it through `appliance.collections.cloud_keypairs.all()[0]`?

Reply:
With ec2  and appliance.collections.cloud_keypairs.all()[0], the navigation took really long since ec2 has ~750 keypairs  and it was navigating through all the pages. So, I decided to use view.entities.get_first_entity().data['name']
 instead.